fix word-break

hjanuschka opened the pull request:
implements suggestion from https://github.com/getsentry/sentry/issues/7452

billyvg commented on src/sentry/static/sentry/less/type.less:
```diff
@@ -178,7 +178,7 @@ code {
 
 pre {
-  word-break: keep-all;
+  word-break: break-all;
```
Instead of changing this for all `pre` tags, can we scope it to `.event-details-container` in (`src/sentry/static/sentry/less/group-detail.less`)?

hjanuschka commented:
@billyvg good point.

i addressed the feedback, i did scope it somwhere else, as it seems to be otherwise not working.


i also removed the `max-height` on the `td` - otherwise it would be cut.


![bildschirmfoto 2018-07-11 um 09 26 54](https://user-images.githubusercontent.com/2891702/42556738-da3c0af6-84ec-11e8-822a-69761443e926.png)


the screenshot above has a very long string without breakable spaces, and it renders fine!


thx for your time, and please let me know if you'd need anything else adressed

billyvg commented:
Thank you @hjanuschka, this looks great!
